tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-export ROOTDIR=$(git rev-parse --show-toplevel)
-
+ROOTDIR=$(git rev-parse --show-toplevel)
+export ROOTDIR
```

Comment:
have you tried it? I think it won't work.

Reply:
Yup, Tested locally works

Reply:
Sorry, my machine got weird behavior. Previously it requires `export ROOTDIR=$ROOTDIR`. Now it can run.